pages/_app.tsx

```diff
@@ -8,9 +8,10 @@ import { ThemeProvider as MUIThemeProvider } from "@mui/material";
 import { SWRConfig } from "swr";
 import { usePreferredTheme } from "../utils/hooks/usePreferredTheme";
 import { Inter } from "@next/font/google";
```

Comment:
`yarn remove @next/font`

```suggestion
import { Inter } from "next/font/google";
```